persistence/player_repository.go

```diff
@@ -31,26 +31,33 @@ func (r *playerRepository) Put(p *model.Player) error {
 	return err
 }
 
+func (r *playerRepository) baseRequest(options ...model.QueryOptions) SelectBuilder {
```

Comment:
For consistency sake, let's call this method `selectPlayer` (see other repositories like album, artist..)